### PR TITLE
Feat/habitat activites

### PR DESCRIPTION
**Pull request by kealanparr**

### CODE REVIEW

File: packages/web-service/src/pages/habitat/active-entrances/habitat-active-entrances.njk [hunk 1]
```diff
@@ -1,37 +1,30 @@
 {% extends "standard-form.njk" %}
 
-{% from "error-summary.njk" import errorSummary %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 
 {% set errorMap = {
         'habitat-active-entrances': {
-          'no-radio-selected': { ref: '#habitat-reopen', text: 'You have not selected a choice.' }
```
Comment (GrahamMWillis) [inline packages/web-service/src/pages/habitat/active-entrances/habitat-active-entrances.njk:10]: I believe that the GDS standard for single sentences with no follow on text is to omit the full-stop (.).

File: packages/web-service/src/pages/habitat/grid-ref/habitat-grid-ref.js [hunk 2]
```diff
@@ -1,6 +1,38 @@
...
+      message: 'Error: no grid ref has been sent',
+      path: [habitatGridKey],
+      type: 'no-reference-sent',
+      context: {
+        label: habitatGridKey,
+        value: 'Error',
+        key: habitatGridKey
+      }
+    }], null)
+  } else if (/[a-zA-Z]{2}\d{6}/.test(gridRef) === false) {
```
Comment (GrahamMWillis) [inline packages/web-service/src/pages/habitat/grid-ref/habitat-grid-ref.js:22]: Not sure why you are doing this. Just use the Joi.object in the validator arg as before

**Review by GrahamMWillis — CHANGES_REQUESTED** [review]

File: packages/web-service/src/pages/habitat/active-entrances/habitat-active-entrances.njk (hunk 1 above)
Comment (kealanparr) [reply packages/web-service/src/pages/habitat/active-entrances/habitat-active-entrances.njk:10]: Thanks for explaining, resolved

File: packages/web-service/src/pages/habitat/grid-ref/habitat-grid-ref.js (hunk 2 above)
Comment (kealanparr) [reply packages/web-service/src/pages/habitat/grid-ref/habitat-grid-ref.js]: Changed it

**Review by GrahamMWillis — APPROVED** [review]

**Pull request merged**